src/python/grpcio/commands.py

```diff
@@ -74,3 +75,29 @@ def run(self):
       conf_file.write(_CONF_PY_ADDENDUM)
     sphinx.main(['', os.path.join('doc', 'src'), os.path.join('doc', 'build')])
 
+
+class BuildProjectMetadata(setuptools.Command):
+  """Command to generate project metadata in a module."""
+
+  description = ''
+  user_options = []
+
+  def initialize_options(self):
+    pass
+
+  def finalize_options(self):
+    pass
+
+  def run(self):
+    print os.getcwd()
+    with open('grpc/_grpcio_metadata.py', 'w') as module_file:
+      module_file.write('__version__ = """{}"""'.format(
```

Comment:
A vague notion of not wanting to ever parse quote escapes and laziness to look up something that implements quote escapes. In short: entirely arbitrary.